src/sentry/coreapi.py

```diff
@@ -369,12 +380,17 @@ def parse_client_as_sdk(self, value):
         }
 
     def should_filter(self, project, data, ip_address=None):
-        # TODO(dcramer): read filters from options such as:
-        # - ignore errors from spiders/bots
-        # - ignore errors from legacy browsers
         if ip_address and not is_valid_ip(ip_address, project):
             return True
 
+        release = data.get('release')
+        if release and not is_valid_release(release, project):
+            return True
+
+        error_message = data.get('sentry.interfaces.Message', {}).get('message')
+        if error_message and not is_valid_error_message(error_message, project):
```

Comment:
This is being applied to the _unformatted_ message, e.g.

```
    "sentry.interfaces.Message": {
        "formatted": "rule_notify.process_error",
        "message": "%s.process_error",
        "params": [
            "'rule_notify'"
        ]
```

If you tried to apply this to `rule_*.process_error`, you'd expect it to work based on what is displayed in the UI, but it won't have any effect here.